app/user_api/spec.yaml

```diff
@@ -70,3 +70,19 @@ paths:
       responses:
         "200":
           description: User deleted
+  /user/test_authenticate:
+    get:
+      x-openapi-router-controller: app.user_api.user_api
+      operationId: test_authenticate
+      tags:
+        - User
+      summary: "Authenticate using a JWT token"
+      responses:
+        "200":
+          description: Successful authentication, return user email"
+          content:
+            'text/plain':
+              schema:
+                type: string
+      security:
+      - jwt: []
```

Comment:
This is processed on the front-end by calling the firebase auth apis. We don't need a "login" or authenticate method on the backend.

Reply:
This is mainly a template/test endpoint. It doesn't do anything except return the user's email back to you so it can be removed down the line.

Reply:
I see...

Reply:
But if this is a test, why are we merging it with the working codebase?